lib/kino_bumblebee/task_cell.ex

```diff
@@ -1308,16 +1310,39 @@ defmodule KinoBumblebee.TaskCell do
 
         Kino.listen(form, fn %{data: %{audio: audio}} ->
           if audio do
-            Kino.Frame.render(frame, Kino.Text.new("Running..."))
-
             audio =
               audio.data
               |> Nx.from_binary(:f32)
               |> Nx.reshape({:auto, audio.num_channels})
               |> Nx.mean(axes: [1])
 
-            %{results: [%{text: generated_text}]} = Nx.Serving.run(serving, audio)
-            Kino.Frame.render(frame, Kino.Text.new(generated_text))
+            Kino.Frame.render(frame, Kino.Text.new("(Start of transcription)", chunk: true))
+
+            for chunk <- Nx.Serving.run(serving, audio) do
+              [start_mark, end_mark] =
+                for seconds <- [chunk.start_timestamp_seconds, chunk.end_timestamp_seconds] do
+                  seconds = round(seconds)
+
+                  minutes =
+                    seconds
+                    |> div(60)
+                    |> Integer.to_string()
+                    |> String.pad_leading(2, "0")
+
+                  seconds =
+                    seconds
+                    |> rem(60)
+                    |> Integer.to_string()
+                    |> String.pad_leading(2, "0")
+
+                  minutes <> ":" <> seconds
```

Comment:
```suggestion
                  seconds
                  |> round()
                  |> Time.from_seconds_after_midnight()
                  |> Time.to_string()
```

WDYT?

Reply:
It does include hour as well, but it is less code, so I thought i would suggest

Reply:
Oh fantastic, we can do `|> String.replace_prefix("00:", "")`, wdyt?

Reply:
Up to you, I would just keep it with hours and everything :) the less code, the better